src/services/ExplorerService/fetcher.ts

```diff
@@ -254,6 +254,11 @@ export const apiFetcher = {
   fetchBitcoinAddresses: (address: string) =>
     requesterV2.get(`bitcoin_addresses/${address}`).then(res => toCamelcase<BitcoinAddresses>(res.data)),
 
+  fetchBitcoinAddressesUDTAccounts: (address: string) =>
```

Comment:
The name could be more intuitive, such as `fetchUDTAccountsByBtcAddress`, the `address` is singular, so `BitcoinAddresses` should be `BitcoinAddress`

Reply:
It's different. `bitcoin_addresses/{address}/udt_accounts` follows RESTful which is semantic. `UDT_Account` is a resource owned by a `bitcoin_address`. `BitcoinAddressesUDTAccounts` is simply a combination of nouns.